clang/runtime/dpct-rt/include/dpct/blas_gemm_utils.hpp

```diff
@@ -133,7 +134,7 @@ class matmul_desc_t {
   };
 
   matmul_desc_t(compute_type compute_type, library_data_t scale_type)
-      : _compute_type(compute_type), _scale_type(scale_type) {}
+      : _compute_type(compute_type), _scale_type(scale_type), _bias_type(bias_type) {}
```

Comment:
There is no `bias_type` in the constructor parameter, so you can give a default value directly:
```suggestion
      : _compute_type(compute_type), _scale_type(scale_type), _bias_type(some default value) {}
```

Reply:
Ok Wiill modify this

Reply:
I changed to library_data_t  for bias_type (similar to scalar type).
@zhiweij1  could you please suggest ? Thanks

Reply:
71bac7a is correct.

Reply:
Yes fixed, thanks